### PR TITLE
fix trainable readme links

### PR DESCRIPTION
**Pull request by eladc-git**

### CODE REVIEW

File: model_compression_toolkit/quantizers_infrastructure/keras/README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ Trainable quantizer can be Weights Quantizer or Activation Quantizer.
...
+   - Implement [`initialize_quantization`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) where you can define your parameters for the quantizer.
+   - Implement [`__call__`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method to quantize the given inputs while training. This is your custom quantization itself. 
+   - Implement [`convert2inferable`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method. This method exports your quantizer for inference (deployment). For doing that you need to choose one of our Inferable Quantizers ([Inferable Quantizers](https://github.com/sony/model_optimization/tree/main/model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/README.md)) according to target when implement `convert2inferable`, and set your learned quantization parameters there.
```
Comment (reuvenperetz) [inline model_compression_toolkit/quantizers_infrastructure/keras/README.md:15]: I think the link should be to the directory of the quantizers, no?

File: model_compression_toolkit/quantizers_infrastructure/pytorch/README.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ Using this base class makes it simple to implement new quantizers for training a
...
+   - Implement [`initialize_quantization`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) where you can define your parameters for the quantizer.
+   - Implement [`__call__`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method to quantize the given inputs while training. This is your custom quantization itself. 
+   - Implement [`convert2inferable`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) method. This method exports your quantizer for inference (deployment). For doing that you need to choose one of our Inferable Quantizers ([Inferable Quantizers](https://github.com/sony/model_optimization/tree/main/model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/README.md)) according to target when implement `convert2inferable`, and set your learned quantization parameters there.
```
Comment (reuvenperetz) [inline model_compression_toolkit/quantizers_infrastructure/pytorch/README.md:15]: Same

Comment (eladc-git) [reply model_compression_toolkit/quantizers_infrastructure/pytorch/README.md:15]: No. to the readme.

File: model_compression_toolkit/quantizers_infrastructure/keras/README.md (hunk 1 above)
Comment (eladc-git) [reply model_compression_toolkit/quantizers_infrastructure/keras/README.md:15]: No. to the readme.

**Review by reuvenperetz — APPROVED** [review]

**Pull request merged**